driver/src/main/java/org/neo4j/driver/v1/types/Duration.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2002-2018 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.neo4j.driver.v1.types;
+
+import java.time.temporal.TemporalAmount;
+
+import org.neo4j.driver.v1.Values;
+import org.neo4j.driver.v1.util.Immutable;
+
+/**
+ * Represents temporal amount containing months, days, seconds and nanoseconds of the second. A duration¬ can be negative.
+ * <p>
+ * Value that represents a duration can be created using {@link Values#duration(long, long, long, long)} method.
+ */
+@Immutable
+public interface Duration extends TemporalAmount
```

Comment:
Maybe `IsoStandardDuration`?

Reply:
Renamed to `IsoDuration` in https://github.com/neo4j/neo4j-java-driver/pull/474/commits/bb31fc02532fc16ac9d164fc26ed622974d74a0b